### DISCUSSION TITLE
Student Developer Pack Request got Expired. How to reapply?

### DISCUSSION THREAD

**Discussion by pro402** [community/community#142787] [GitHub Education]
### Hi everyone,

I have now check that the status which is causing trouble as i an trying to get the student pack   
![image.png](https://github.zendesk.com/attachments/token/4EZcnZ3se4U8GYAx5wMSgX46S/?name=image.png)
i am not able to do it due to an issue as my email is a gmail id(i dont have a student id)However i want to resubmit my student details like the documents. 
![image.png](https://github.zendesk.com/attachments/token/lbSnSFIpJAAYhwwtxbXUoTR9x/?name=image.png)
But i am unable to do so due to this issue in the image i am not able to proceed further.Help me create a new application

**Comment by moranenzo** [comment]
<p>Hi there!</p>

<p>Thanks for reaching out about your expired Student Developer Pack application! I understand how frustrating this can be, especially without a student ID.</p>

<p>Unfortunately, reapplying can sometimes be tricky without proper documentation. I recommend contacting GitHub Education support directly for assistance. They can provide guidance on how to resubmit your application or help resolve any issues with your email.</p>

<p>Good luck, and feel free to ask if you have any more questions!</p>

**Reply by pro402** [reply]
Thanks a lot i have tried to create a ticket but there is another issue with it.

**Reply by pro402** [reply]
![image](https://github.com/user-attachments/assets/04111e28-88fa-4044-bb03-9d80df9a9dd0)

**Reply by moranenzo** [reply]
Yeah i see, if you only have the free version of github, you can't open a ticket with the staff, you just can open community tickets. That's what your error message means. I am sorry but i think you don't haev any clue to solve your issue.

If my answer helped you don't forget to put the tag "Answered" on your ticket.

Best regards,
Enzo

**Reply by pro402** [reply]
"Unfortunately, reapplying can sometimes be tricky without proper documentation. I recommend contacting GitHub Education support directly for assistance. They can provide guidance on how to resubmit your application or help resolve any issues with your email."

How to connect directly to the GitHub Education Support.

**Reply by moranenzo** [reply]
To contact GitHub Education support, you can use the following steps:

**GitHub Support Page:** Go to [GitHub Education Support](https://support.github.com/contact?tags=education) where you can submit a support ticket. Make sure to select "Education" as the topic for the inquiry to get routed to the correct team.

**Email GitHub Education:** You can also directly email GitHub Education support at [education@github.com](mailto:education@github.com) for inquiries related to GitHub Student Developer Pack, classroom issues, or verification.

**GitHub Education Community Forum:** If you’re looking for peer advice or common issue resolutions, visit the GitHub Education Community Forum, where GitHub staff and experienced users often provide answers and guidance.
